src/components/Book.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
 import Proptypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
```

Comment:
- [ ] Kindly this method comes from `'../redux/books/Books'` Books not books, to solve these errors.

![image](https://user-images.githubusercontent.com/35535130/163198764-3849013c-a5b8-4a63-b2ac-af19736e4ee0.png)